.github/workflows/e2e-test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       fail-fast:     false
       matrix:
         # Min WooCommerce | L-2 | L(atest) | beta
-        woocommerce:   [ '4.6.0', '5.9.0', '6.1.1', 'beta' ]
+        woocommerce:   [ '4.7.0', '6.0.0', '6.2.0', 'beta' ]
```

Comment:
Are we sure we want to replace `5.9.0` with `6.0.0` here? I think we should keep the `5.x.x` branch?

Reply:
When the previous PR was merged (#3716), the latest version was `6.1.1`, and the L-2 was `5.9.0`. Now the latest is `6.2.0`, so I had to also bump the L-2 to be 2 versions behind it.

So I basically just followed the previous PR. Does that make sense?